src/IconButton/IconButton.js

```diff
@@ -200,9 +223,16 @@ class IconButton extends Component {
     const styles = getStyles(this.props, this.context);
     const tooltipPosition = tooltipPositionProp.split('-');
 
+    const hovered = (this.state.hovered || this.state.isKeyboardFocused) && !disabled;
+
+    const mergedRootStyles = Object.assign(
+      styles.root,
+      {backgroundColor: hovered ? hoverColor : transparent},
```

Comment:
just tested, ok the changing colors on mouse leave still works with this new code:
```
    const mergedRootStyles = Object.assign(
      styles.root,
      hovered ? {backgroundColor: hoverColor} : {},
      style
    )
```
is it better like that, without setting backgroundColor (removed _transparent_) as you suggested ?